engine/execution/messages.go

```diff
@@ -17,18 +18,19 @@ type ComputationOrder struct {
 }
 
 type ComputationResult struct {
-	ExecutableBlock    *entity.ExecutableBlock
-	StateSnapshots     []*delta.SpockSnapshot
-	StateCommitments   []flow.StateCommitment
-	Proofs             [][]byte
-	Events             []flow.EventsList
-	EventsHashes       []flow.Identifier
-	ServiceEvents      flow.EventsList
-	TransactionResults []flow.TransactionResult
-	ComputationUsed    uint64
-	StateReads         uint64
-	TrieUpdates        []*ledger.TrieUpdate
-	ExecutionDataID    flow.Identifier
+	ExecutableBlock        *entity.ExecutableBlock
+	StateSnapshots         []*delta.SpockSnapshot
+	StateCommitments       []flow.StateCommitment
+	Proofs                 [][]byte
+	Events                 []flow.EventsList
+	EventsHashes           []flow.Identifier
+	ServiceEvents          flow.EventsList
+	TransactionResults     []flow.TransactionResult
+	ComputationUsed        uint64
+	ComputationIntensities meter.MeteredComputationIntensities
```

Comment:
Can you check if these (+memory) could be packaged into their own struct called `MeteringResults`?